x/fungible/types/evm_error_message.go

```diff
@@ -0,0 +1,42 @@
+package types
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	MethodKey       = "method"
+	ContractKey     = "contract"
+	ArgsKey         = "args"
+	ErrorKey        = "error"
+	RevertReasonKey = "revertReason"
```

Comment:
```suggestion
	RevertReasonKey = "revert_reason"
```

Reply:
I used camel case here to be consistent with other constants defined in the core logic. 
Is the reason you want to do more to snake case that this is going to be used by external users ? If so , I think moving to snake case makes sense , but lmk if there is a different reason for the change

Reply:
I just think snake case looks better in user message